src/sirinternal.c

```diff
@@ -197,7 +203,11 @@ bool _sir_init(sirinit* si) {
 
     _SIR_UNLOCK_SECTION(SIRMI_CONFIG);
 
+#if defined(SIR_NO_TEXT_STYLING) && defined(SIR_NO_SYSTEM_LOGGERS)
+    _sir_selflog("initialized successfully");
```

Comment:
not needed; `init` is initialized to true, so it will say successfully already

Reply:
Yes, but PVS-Studio will complain if both SIR_NO_TEXT_STYLING and SIR_NO_SYSTEM_LOGGERS are defined that the condition is always true, because there is no way that init could be set to false.  

But I can do it that way and suppress the warning, if you prefer.